app/controllers/weight_estimator_controller.rb

```diff
@@ -1,5 +1,5 @@
 class WeightEstimatorController < ApplicationController
   def index
-    @household_good_categories = HouseholdGoodCategory.all.order(:name)
+    @household_good_categories = HouseholdGoodCategory.includes(:household_goods).all
```

Comment:
Active Record's [`include` method](http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-includes) will… include… the specified relationships when it grabs results from the database.

Prior to the changes in this PR, the code `category.household_goods.order(:name)` (in the view, `app/views/weight_estimator/_household_goods.html.erb`) was hitting the database on every iteration of the loop to grab `category`'s `household_goods`.